tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -346,7 +340,7 @@ The selector matches the labels set by the Helm chart on the ingress-nginx pods,
     ```
 
 5. Add the creation of a DNS record to direct the external traffic to the available load balancers. We assume that you have an internal or external domain configured in your account with a `scw` zone, as described in the prerequisites part at the beginning of this section. In the code below, we use `your-domain.tld` as placeholder, so you need to replace it with your domain name. If you want to use a bare-domain, remove the subdomain parameter from the `scaleway_domain_zone` resource.
-In this configuration we use the `http_service` block to check that the `ingress-nginx` service is up and running via each load balancer on the two AZs. We retrieve the IPs of the LoadBalancers via the "EXTERNAL-IP" exposed by the Kubernetes services. The DNS record "ingress" under your domain `scw.your-domain.tld` will resolve to all the IPs of the healthy load balancers, via the "all" strategy. So in case of the loss of one AZ, the DNS record will automatically be updated to resolve only to the IP of the healthy load balancer, and the traffic will be dispatched to the healthy nodes of the other AZs.
+In this configuration we use the `http_service` block to check that the `ingress-nginx` service is up and running via each load balancer on the two AZs. We retrieve the IPs of the LoadBalancers via the "EXTERNAL-IP" exposed by the Kubernetes services. The DNS record "ingress" under your domain `scw.your-domain.tld` will resolve to all the IPs of the healthy load balancers, via the "all" strategy. So in case of the loss of one AZ, the DNS record will automatically be updated to resolve only to the IP of the healthy load balancer, and the traffic will be dispatched to the healthy nodes of the other AZs. The host used in the "url" must not match any ingress host defined in the cluster, otherwise the "/healthz" endpoint will not work.
```

Comment:
```suggestion
In this configuration we use the `http_service` block to check that the `ingress-nginx` service is up and running via each load balancer on the two AZs. We retrieve the IPs of the LoadBalancers via the "EXTERNAL-IP" exposed by the Kubernetes services. The DNS record "ingress" under your domain `scw.your-domain.tld` will resolve to all the IPs of the healthy load balancers, via the "all" strategy. So in case of the loss of one AZ, the DNS record will automatically be updated to resolve only to the IP of the healthy load balancer, and the traffic will be dispatched to the healthy nodes of the other AZs.
<Message type="important">
  The host used in the "url" must not match any ingress host defined in the cluster, otherwise the "/healthz" endpoint will not work.
</Message>
```